### DISCUSSION TITLE
How to transition a `motion.div` between containers with `overflow-y: auto` set

### DISCUSSION THREAD

**Discussion by wilson-webdev** [motiondivision/motion#1658] [Q&A]
I have a trello-style column layout. Each item is a `motion.div` with a unique `layoutId` prop. As you can see from the below videos, when `overflow-y: auto` is set, you cannot see the items transitioning between columns. Each column must be individually scrollable, so I can't just set the overflow on the outer `div`. Some tips would be great! 😃 

### CodeSandbox

https://codesandbox.io/s/framer-motion-2-layout-animations-forked-n41fe2?file=/src/App.js

### `overflow-y: auto`

https://user-images.githubusercontent.com/24278706/184231806-574fedac-b6dc-4766-8293-0bf43186c0f6.mov

### `overflow-y: initial`

https://user-images.githubusercontent.com/24278706/184232997-321ed747-095c-4d54-8409-0599440b2300.mov

**Comment by marcinskubis** [comment]
I got a same problem. Did you manage to come up with a solution?

**Reply by wilson-webdev** [reply]
Unfortunately not, sorry! I was only experimenting with some features, hopefully you can find a solution!

**Reply by alexsafayan** [reply]
@marcinskubis did you find a solution?

**Reply by marcinskubis** [reply]
> @marcinskubis did you find a solution?

Nope 😕

**Reply by alexsafayan** [reply]
React's `createPortal` looks promising

**Reply by altrugon** [reply]
Has anybody found a solution for this?
I did try `createPortal()` but ended up in the same situation. The animated container is only visible when entering the destination container that has overflow set, or its parent has it.

**Comment by alexsafayan** [comment]
Framer was able to do it... answer hopefully coming soon

https://x.com/alexsafayan/status/1970165507325391227

**Comment by alexsafayan** [comment]
https://github.com/motiondivision/motion/issues/2491
